backend/mock/service/feedbackmock/feedbackmock.go

```diff
@@ -0,0 +1,18 @@
+package feedbackmock
+
+import (
+	"context"
+
+	feedbackv1 "github.com/lyft/clutch/backend/api/feedback/v1"
+	"github.com/lyft/clutch/backend/service/feedback"
+)
+
+type svc struct{}
+
+func (s svc) SubmitFeedback(ctx context.Context, id string, feedback *feedbackv1.Feedback, metadata *feedbackv1.FeedbackMetadata) error {
+	return nil
+}
+
+func New() feedback.Service {
```

Comment:
do we want to register this in the mock server for testing?

Reply:
registered it but will follow up in another to add error scenarios.